src/components/MaterialUI/Inputs/InputBase.js

```diff
@@ -86,7 +86,8 @@ const InputBase = ({ inputProps }) => {
 	const placeholder = inputProps?.get(InputBaseProps.propNames.placeholder);
 	const error = inputProps?.get(InputBaseProps.propNames.error);
 	const errorPosition = inputProps?.get(InputBaseProps.propNames.errorPosition);
-	const disabled = inputProps?.get(InputBaseProps.propNames.disabled) || false;
+	const disabled = inputProps?.get(InputBaseProps.propNames.disabled) ?? false;
+	const readOnly = inputProps?.get(InputBaseProps.propNames.readOnly) ?? false;
```

Comment:
so, are we going to use readOnly? previous information was that we are going to use just disabled.. can you please specify some use cases for readonly please?

Reply:
Oleksandr please remove this readOnly property. Thanks